requirements.in

```diff
@@ -25,6 +25,3 @@ scipy
 boto
 boto3
 semantic_version
-# sentiment analysis
-google-cloud
-joblib
```

Comment:
Are you going to also remove [workers/sentiment_scorer.py](https://github.com/fabric8-analytics/fabric8-analytics-worker/blob/master/f8a_worker/workers/sentiment_scorer.py) or not ? (perhaps you just forgot to 'git rm sentiment_scorer.py' ?)

If the file is going to stay there, then I'd just comment out the dependencies in requirements.in, so that we don't install them, but are aware of what's requirements.

In any case you need to regenerate [requirements.txt](https://github.com/fabric8-analytics/fabric8-analytics-worker/blob/master/requirements.txt)

Reply:
removed